features/support/paths.rb

```diff
@@ -247,6 +247,11 @@ def path_to(page_name, *args) # rubocop:disable Metrics/AbcSize, Metrics/Cycloma
       service = provider.services.find_by_name! $2
       polymorphic_path [$1.to_sym, :admin, service]
 
+    when /^the edit page for feature "([^"]+)"$/
+      plan = @provider.service_plans.where(name: "Basic").last
```

Comment:
This steps is not correct.

1. It assumes that the feature belongs to Service plans, and that there is a plan "Basic". That's not right.

2. This leads to a page that does actually have feature edit functionality, but it is NOT how we update features in the admin portal.

This page looks like this:

![image](https://github.com/3scale/porta/assets/1270328/ee2a085b-c938-498c-b98b-f3b8b6414be0)

While in the product we update features thought this modal window:

![image](https://github.com/3scale/porta/assets/1270328/7349ba17-9f23-4ca6-873e-ad4b189e57f6)